lib/index.js

```diff
@@ -71,22 +71,20 @@ function onlyFiles(schemas) {
 
 function getAllObjects(baseQuery)  {
   var allObjects = [];
-  var next = function(startIndex) {
-    if (startIndex > 10000) {
+  var next = function() {
+    if (allObjects.length) {
       baseQuery.greaterThan('createdAt', allObjects[allObjects.length-1].createdAt);
```

Comment:
We do 😃 The query is created in `getObjectsWithFilesFromSchema` with `  query.ascending('createdAt')`  on line 93.